### PR TITLE
*: disable parallel to avoid data race when test

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Problem Summary:

statistics: disable parallel to avoid data race when test.

many tests will change ```domain.RunAutoAnalyze``` that lead to a data race.

### Check List

Tests 

- [x] Unit test

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by hawkingrei** [comment]
@tisonkun @AilinKid @zimulala PTAL

**Review by zimulala — COMMENTED** [review]
Could you indicate in the description where data race occurs?

**Comment by hawkingrei** [comment]
> Could you indicate in the description where data race occurs?

Yes, I have added.

**Review by tisonkun — COMMENTED** [review]
Please try to follow the practice in https://github.com/pingcap/tidb/issues/26022

> When test cases should be run in serial at the top level, move them into `xxx_serial_test.go` file.

otherwise, it decreases the readability of code.

File: infoschema/perfschema/tables_test.go [hunk 1]
```diff
@@ -57,8 +57,6 @@ func TestPerfSchemaTables(t *testing.T) {
 }
 
 func TestTiKVProfileCPU(t *testing.T) {
-	t.Parallel()
```
Comment (Defined2014) [inline infoschema/perfschema/tables_test.go:60]: The common way to fix it is to create a new test file called `xxx_serial_test.go` in same package.

Comment (hawkingrei) [reply infoschema/perfschema/tables_test.go:60]: Done

**Comment by hawkingrei** [comment]
> Please try to follow the practice in #26022
> 
> > When test cases should be run in serial at the top level, move them into `xxx_serial_test.go` file.
> 
> otherwise, it decreases the readability of code.

I have fixed.

**Comment by hawkingrei** [comment]
@tisonkun @Defined2014 PTAL

**Review by Defined2014 — APPROVED** [review]

**Review by lcwangchao — APPROVED** [review]

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**